extensions/amp-bind/amp-bind.md

```diff
@@ -293,3 +321,42 @@ object:
 key_value:
   expr ':' expr
 ```
+
+## Debugging
+
+Testing in development mode (with the URL fragment `#development=1`) is recommended to clearly surface warnings and errors during development.
+
+### Warnings
+
+In development mode, `amp-bind` will issue a warning when the default value of a bound attribute doesn't match its corresponding expression's initial result. This can help prevent unintended mutations caused by changes in other state variables. For example:
+
+```html
+<!-- The element's default value ('def') doesn't match the initial expression result ('abc'),
+     so a warning will be issued in development mode. -->
+<p [text]="'abc'">def</p>
+```
+
+In development mode, `amp-bind` will also issue a warning for undefined variables and property dereferences. This can also help prevent unintended mutations due to `null` expression results. For example:
```

Comment:
Optional: 

...will issue a warning when dereferencing undefined variables. This can...